### PR TITLE
Documentation of Raspberry Pi ID service

### PR DESCRIPTION
**Pull request by aallan**
Initial documentation for the Raspberry Pi ID service.

There is one bug presenting. On Safari the top bars resize correctly with the additional category.

Screenshot 2023-03-23 at 11 36 43

However in Chrome it doesn't resize properly

Screenshot 2023-03-23 at 11 36 47

Not sure where (in the CSS?) this is breaking?

### CODE REVIEW

**Comment by aallan** [comment]
@nelliemckesson When you get a chance could you take a quick look at what's presumably a CSS bug so we can merge this for @mudge? Thanks!

File: documentation/asciidoc/services/id/2fa.adoc [hunk 1]
```diff
@@ -0,0 +1,45 @@
+= Two-factor authentication
+
+Like most modern web services, the Raspberry Pi ID services supports two-factor authentication (2FA) using a https://en.wikipedia.org/wiki/Time-based_one-time_password[time-based one-time password] (TOTP).
```
Comment (mudge) [inline documentation/asciidoc/services/id/2fa.adoc:3]: Should this be "Raspberry Pi ID _service_" singular?

File: documentation/index.json [hunk 2]
```diff
@@ -158,6 +158,18 @@
...
             ]
         },
+        {
+            "title": "Services",
+            "path": "services",
+            "subitems": [
+                {
+                    "title": "Raspberry Pi ID",
+                    "description": "Our identity service",
+                    "image": "full-sized/Getting-Started.png",
```
Comment (mudge) [inline documentation/index.json:168]: Does this mean Raspberry Pi ID will have the same icon as "Getting Started"?

File: documentation/asciidoc/services/id/signing_up.adoc [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+If your password matches, the service will send an email to the email address you entered. Go to your inbox and open the email, then click on the "Verify email" button in the email.
+
+image::images/verify_identity.png[width="80%"]
+
+You will be signed into the identity service.
+
+image::images/signed_in_identity.png[width="80%"]
+
+After creating a Raspberry Pi ID, you can log in to Raspberry Pi services using the "Sign in with Raspberry Pi" button to sign in with your Raspberry Pi ID. After logging in with your identity, you may be asked to authorise the service to use your Raspberry Pi ID.
```
Comment (mudge) [inline documentation/asciidoc/services/id/signing_up.adoc:23]: Should we drop this last sentence beginning "After logging in" given we consider official Raspberry Pi client applications of Raspberry Pi ID (e.g. Raspberry Pi Events) to be trusted (so users will not see a prompt to authorise access to their information in ID)?

File: documentation/asciidoc/services/id/images/create_account_identity.png
Comment (mudge) [inline documentation/asciidoc/services/id/images/create_account_identity.png:1]: Minor detail but the example email address used in the other screenshots is `test@raspberrypi.com` but this screenshot shows `test@raspberry.com`.

Comment (aallan) [reply documentation/asciidoc/services/id/images/create_account_identity.png:1]: Doh! 🤦‍♂️

Comment (aallan) [reply documentation/asciidoc/services/id/images/create_account_identity.png:1]: Fixed in 3628ed5.

File: documentation/asciidoc/services/id/signing_up.adoc [hunk 4]
```diff
@@ -0,0 +1,23 @@
+== Signing up to Raspberry Pi ID 
+
+To use Raspberry Pi services, you will first have to create a Raspberry Pi ID account using our https://id.raspberrypi.com[identity service]. This account will be your gateway to the Raspberry Pi website and other services.
```
[the remark this answers is not in the document]
Comment (aallan) [reply documentation/asciidoc/services/id/signing_up.adoc]: Fixed in fddf34a.

File: documentation/asciidoc/services/id/signing_up.adoc (hunk 3 above)
Comment (aallan) [reply documentation/asciidoc/services/id/signing_up.adoc]: Fixed in fddf34a.

**Comment by mudge** [comment]
> @nelliemckesson When you get a chance could you take a quick look at what's presumably a CSS bug so we can merge this for @mudge? Thanks!

One thing we could do here is switch the tabs to use [flexbox](http://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Flexible_Box_Layout/Basic_Concepts_of_Flexbox), e.g.

```css
#tab-container {
  display: flex;

  /* Set a maximum width to match the rest of the site and centre it horizontally. */
  max-width: 1300px;
  margin-left: auto;
  margin-right: auto;
}

#tab-container li {

  /* Tell all tabs to be the same width and to grow and shrink as needed. */
  flex: 1 1 0%;
}
```

This would give you something like the following on narrow screens:

image

And this on wide ones:

image

However, you'd want to switch `#tab-container` to `flex-direction: column` on the narrowest screens (when the tabs can't possibly fit) so they start to stack vertically, e.g.

image

**Comment by aallan** [comment]
This all looks fine with me!

**Comment by aallan** [comment]
LGTM!

**Pull request merged**